### ISSUE TITLE
Error on npm install

### ISSUE THREAD

**Issue by luastoned** [andrew-1135/snowpack-start#1]
Hello, I tried your starter kit but it errors before finishing the installation.

```
- Creating a new Snowpack app in /code/node/starter-kits/snowpack

- Installing package dependencies. This might take a couple of minutes.
node:internal/errors:415
    ErrorCaptureStackTrace(err);
    ^

Error: Command failed with ENOENT: npm install react@17 react-dom@17
spawnSync npm install ENOENT
    at Object.spawnSync (node:internal/child_process:1084:20)
    at Object.spawnSync (node:child_process:668:24)
    at Function.module.exports.sync (/code/.npm/_npx/daf316dc1912ab5b/node_modules/execa/index.js:167:25)
    at installPackages (/code/.npm/_npx/daf316dc1912ab5b/node_modules/snowpack-start/src/index.js:362:11)
    at main (/code/.npm/_npx/daf316dc1912ab5b/node_modules/snowpack-start/src/index.js:579:3)
    at processTicksAndRejections (node:internal/process/task_queues:94:5) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawnSync npm install',
  path: 'npm install',
  spawnargs: [ 'react@17', 'react-dom@17' ],
  originalMessage: 'spawnSync npm install ENOENT',
  shortMessage: 'Command failed with ENOENT: npm install react@17 react-dom@17\n' +
    'spawnSync npm install ENOENT',
  command: 'npm install react@17 react-dom@17',
  exitCode: undefined,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: '',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
npm ERR! code 1
npm ERR! path /code/node/starter-kits
npm ERR! command failed
npm ERR! command sh -c snowpack-start
```

**Comment by andrew-1135** [comment]
Thanks for the report. I think I know what the issue is, working on a fix now.

**Comment by andrew-1135** [comment]
Just published, try the newest version `1.0.0-beta.3`.

**Comment by luastoned** [comment]
Thanks a bunch!
